testing/error.go

```diff
@@ -0,0 +1,23 @@
+package testing
+
+func IsErrorEqual(a, b error) bool {
+	isANil := a == nil
+	isBNil := b == nil
+
+	// If both are nil then true
+	if isANil && isBNil {
```

Comment:
```suggestion
	if a == nil && b == nil {
```
Anyways we are doing a comparison, we can as well avoid creating new variables

Reply:
yes, done